gitops/apps/aligner-api/tests/assert-manifests.sh

```
@@ -16,11 +16,15 @@ for overlay in normal degraded maintenance; do
   rg -q '^kind: PodDisruptionBudget$' "$render_dir/$overlay.yaml"
   rg -q '^kind: HTTPRoute$' "$render_dir/$overlay.yaml"
   rg -q 'aligner-api-default-deny' "$render_dir/$overlay.yaml"
-  ! rg -q '^kind: Namespace$|image: .*:latest' "$render_dir/$overlay.yaml"
+  ! rg -q '^kind: Namespace$' "$render_dir/$overlay.yaml"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)